packages/styles/src/functions/_color.scss

```diff
@@ -10,8 +10,13 @@
   }
   $percent: color.alpha($color) * 100;
   $opaque: color.adjust($color, $alpha: 1);
+  $mixed: color.mix($opaque, $background, $percent);
```

Comment:
What's the result of `color.mix`? It might be enough to just return this color instead of explicitly converting it to rgb.

Reply:
I think it mixes the fg and the bg colors together and produces a full-tone color.